app/domain/authentication/authn_k8s/kubectl_exec.rb

```diff
@@ -20,12 +21,19 @@ module AuthnK8s
                   body
                   stdin )
     ) do
+
+      extend Forwardable
+      def_delegators :@k8s_object_lookup, :kubectl_client
+
+      COPY_FILE_LOG_FILE = "/tmp/conjur_copy_file.log"
```

Comment:
I think the directory should be evaluated on the machine itself under `${TMPDIR:=/tmp}` as this is more POSIX-compliant.